fizzbuzz_test.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+from fizzbuzz import fizzbuzz
+
+
+class fizzbuzzTest(unittest.TestCase):
+
+    def test_non_list_inputs(self):
+        self.assertEqual(fizzbuzz(5, 8), "Invalid input")
+
+    def test_non_list_input(self):
+        self.assertEqual(fizzbuzz([2, 3, 4], 7), "Invalid input")
+
+    def test_fizz(self):
+        self.assertEqual(fizzbuzz([1, 3, 4], ['a', 'b', 'c']), "fizz")
+
+    def test_buzz(self):
+        self.assertEqual(fizzbuzz([4, 5, 4], [3, 4]), "buzz")
+
+    def test_fizz_buzz(self):
+        self.assertEqual(fizzbuzz([1, 2, 3, 3, 4, 5, 5, 5, 5, 6], [7, 7, 4, 5, 9]), "fizzbuzz")
+
+    def test_other_total(self):
+        self.assertEqual(fizzbuzz([4, 5, 6], [3]), 4)
+
+    def test_both_empty(self):
```

Comment:
This test looks like it should test for what happens when both lists are empty but its not the case

Reply:
For this case, I think we shall need to discuss it when I come coz it looks like even if one of them is empty, it should go ahead and compute the combined length and return the required result.
May be, I can add the empty test condition in the fizzbuzz file and then add its test in the test file.
Otherwise thanks @dennisja